src/crystalla/ndarray.cr

```diff
@@ -0,0 +1,523 @@
+require "./ndarray/builders"
+require "./transformations/pca"
+require "./utils/random_gaussian"
+require "./ndarray/functions"
+
+module Crystalla
+
+
+  # class Dtype(T)
```

Comment:
Let's remove these comments unless you have a very good reason to keep them here!